test/run

```diff
@@ -76,6 +77,7 @@ function test_connection() {
   fi
   echo "Logs:"
   docker logs $(ct_get_cid $name)
+  ct_check_testcase_result 1
```

Comment:
This does not make sense AFAIK. docker logs are working properly in case of presence CID file.

Reply:
This is only a workaround to set TESTCATE_RESULT to 1, because I do not want to set it directly from the test function.

Please note, that the code is not
```
ct_check_testcase_result $?
```
but
```
ct_check_testcase_result 1
```